website/routes.py

```diff
@@ -173,6 +173,8 @@ def make_url_map(app):
 
     process_rules(app, [
 
+        Rule('/home/', 'get', website_views.home, OsfWebRenderer('home.mako')),
```

Comment:
We need to maintain backwards compat for all web routes, so we can't rename.

How about 

`/dashboard/` -> New dashboard
`/dashboard/projects/` -> My projects

?